test/show.jl

```diff
@@ -760,6 +793,131 @@ end"""
         \$a + \$b
     end
 end"""
+@test repr(Meta.parse(
+"""macro m(a, b)
+    quote
+        \$a + \$b
+    end
+end""")) ==
+"""
+:(macro m(a, b)
+      #= none:2 =#
```

Comment:
If it helps, you can strip line number nodes with `Base.remove_linenums!` so they don't need to be included in the reference string here.

Reply:
Thanks for the suggestion.